.cypress/integration/5_trace_analytics_services.spec.js

```diff
@@ -312,10 +291,8 @@ describe('Testing switch mode to jaeger', () => {
   });
 
   it('Verifies traces links to traces page with filter applied', () => {
-    cy.wait(delay);
+    cy.get('.euiTableRow').should('have.length.lessThan', 7);//Replaces Wait
```

Comment:
2 or 3 is immediately understandable.  7 ?  Is that just the injected trace data?

Reply:
The previous state has 7 listings, my thought process is to detect the minimal change to indicate the previous action was applied before running get(). This can be lowered but wanted to use the assertions just to check that the action had been applied while keeping reference to what the previous state had, for going back through the snapshots.